test/helper.js

```diff
@@ -0,0 +1,10 @@
+function once (obj, event, fn) {
```

Comment:
Please add a `'use strict'` to the top of this file and [jsdoc](https://jsdoc.app) to this function.